README.md

```diff
@@ -42,7 +42,7 @@ Installation steps:
     * run `rake db:migrate`
     * OR
     * in MySQL, `drop database XXX;` for each database in `config/database.yml` and then try `rake db:setup` again
-* `rake db:seed` to populate it with initial data
+* `rake db:seed` to populate it with initial dummy data
```

Comment:
I believe it's a wiki page and research note and user for devs to see --
not automated testing data, but not real data either. I said dummy data in
order to distinguish from data in the test db and so people know it's fake.

re: AGPL, I don't disagree, but we've always said GPLv3, so that's what the
assumption's been for folks who've contributed.

Can we cross license, or re-license under AGPL? I folks are OK with AGPL,
can you say so in this thread too? I'm fine with either or both.

On Mon, Feb 23, 2015 at 11:48 AM, Bryan Bonvallet <notifications@github.com>
wrote:

> In README.md
> <https://github.com/publiclab/plots2/pull/235#discussion_r25179385>:
>
> > @@ -42,7 +42,7 @@ Installation steps:
> >      * run `rake db:migrate`
> >      * OR
> >      * in MySQL, `drop database XXX;` for each database in `config/database.yml` and then try `rake db:setup` again
> > -* `rake db:seed` to populate it with initial data
> > +* `rake db:seed` to populate it with initial dummy data
>
> is it dummy data? I thought it was just the most basic data required to
> get up and running. "dummy data" makes it sound like it'd be for running
> tests only, but not for bootstrapping a brand spanking new and potentially
> production-ready system.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/publiclab/plots2/pull/235/files#r25179385>.
>